### PR TITLE
fixed up count on admin and lrs dashboard. Need suggestions!

### PR DESCRIPTION
**Pull request by loongchan**
Hey Ryan,
I fixed the counts on the dashboard for actors, but it's not very well done.  If you have any suggestions on how to improve it, I'd love to make changes to it.

### CODE REVIEW

File: app/locker/data/dashboards/LrsDashboard.php [hunk 1]
```diff
@@ -52,19 +52,40 @@ public function statementCount(){
...
+    
+    $count_array['mbox_sha1sum'] = $this->db->statements->aggregate(
+            array('$match' => $this->getMatch( $this->lrs )),
+            array('$group' => array('_id' => '$statement.actor.mbox_sha1sum')),
+            array('$group' => array('_id' => 1, 'count' => array('$sum' => 1)))
+    );
+    
+    $count_array['account'] = $this->db->statements->aggregate(
+            array('$match' => $this->getMatch( $this->lrs )),
+            array('$group' => array('_id' => '$statement.actor.account.name')),
```
Comment (ryansmith94) [inline app/locker/data/dashboards/LrsDashboard.php]: This needs changing to include `statement.actor.account.homePage` too. You might be able to do:
```php
array('_id' => array(
  'accountName' => '$statement.actor.account.name',
  'accountHomePage' => '$statement.actor.account.homePage'
)
```

File: app/locker/data/dashboards/LrsDashboard.php [hunk 2]
```diff
@@ -52,19 +52,40 @@ public function statementCount(){
    *
    **/
   public function actorCount(){
-
-    $count = $this->db->statements->aggregate(
+    $count_array = ['mbox' => '', 'openid' => '', 'mbox_sha1sum' => '', 'account' => ''];
+    
+    $count_array['mbox'] = $this->db->statements->aggregate(
               array('$match' => $this->getMatch( $this->lrs )),
```
Comment (ryansmith94) [inline app/locker/data/dashboards/LrsDashboard.php]: If possible, please convert the `array(...)` to `[...]` syntax, to maintain consistency and improve readability.

**Comment by loongchan** [comment]
Hey Ryan,
Great comments.  Fixed as requested!

**Comment by ryansmith94** [comment]
Hey @loongchan. Thanks. Awesome! I've just added one last comment before merging this.

**Comment by loongchan** [comment]
Hey Ryan,
Yes, Your suggestion about account query and my understanding of aggregate query function in mongo seem to mesh, so I'm good with it. I also did queries manually to spot check that the results returned was what I expected. 

Best,
Loong

**Comment by ryansmith94** [comment]
Thanks for this @loongchan :)

**Pull request merged**